actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -548,6 +548,8 @@ def current_page?(options)
         request_uri = url_string.index("?") ? request.fullpath : request.path
         request_uri = URI.parser.unescape(request_uri).force_encoding(Encoding::BINARY)
 
+        url_string.sub!(%r{^/(.*)/$},'/\1')
```

Comment:
For now I can only come up with one example: if the `url_string` is `"/"`, chomp will makes it to be `""`. And if you want to prevent this situation you might also need to add another condition for it, so I think using regex to match a certain pattern will be better than just chomp the value.